libs/vscode/tasks/src/lib/cli-task.ts

```diff
@@ -1,5 +1,5 @@
 import { CliTaskDefinition } from './cli-task-definition';
-import { Task, TaskGroup, TaskScope } from 'vscode';
+import { ShellExecution, Task, TaskGroup, TaskScope } from 'vscode';
```

Comment:
Is ShellExecution used somewhere here? I don't see a reference within this file. 

Maybe it has side effects?